### PR TITLE
Ddc test stable

### PR DESCRIPTION
**Pull request by devoncarew**
Test on both dev and stable; fix https://github.com/dart-lang/dartdoc/issues/827.

### CODE REVIEW

**Comment by devoncarew** [comment]
@sethladd

File: appveyor.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 install:
 # TODO: Switch back to chocolatey once 1.12.0 is available.
 #  - choco install -y dart-sdk -version 1.11.1
-  - ps: wget https://gsdview.appspot.com/dart-archive/channels/dev/raw/latest/sdk/dartsdk-windows-x64-release.zip -OutFile dart-sdk.zip
+  - ps: wget https://gsdview.appspot.com/dart-archive/channels/stable/raw/latest/sdk/dartsdk-windows-x64-release.zip -OutFile dart-sdk.zip
```
Comment (sethladd) [inline appveyor.yml]: is there a way to run appveyor on both stable and dev?

**Comment by sethladd** [comment]
lgtm thanks!

File: appveyor.yml (hunk 1 above)
Comment (devoncarew) [inline appveyor.yml]: Don't know; I can investigate.

**Pull request merged**